vendor.conf

```diff
@@ -6,7 +6,7 @@ k8s.io/apiserver release-1.6 https://github.com/kubernetes/apiserver
 #
 github.com/Sirupsen/logrus v0.11.5
 github.com/containers/image b36d6535410088370aaaee7ec8522863b5e43489
-github.com/containers/storage 2c75d14b978bff468e7d5ec3ff8a003eca443209
+github.com/containers/storage master
```

Comment:
:+1: otherwise every time we run vndr, we will end up pulling code from master. Also, it is easier to debug if we know what exact version we have vendored.